streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -66,6 +66,19 @@
      */
     KStream<K, V> filter(final Predicate<? super K, ? super V> predicate);
 
+    /**
+     * Create a new {@code KStream} that consists of all records of this stream which satisfy the given predicate.
+     * All records that do not satisfy the predicate are dropped.
+     * This is a stateless record-by-record operation.
+     *
+     * @param predicate a filter {@link Predicate} that is applied to each record
+     * @param named     the {@link Named} to be used for the processor
```

Comment:
```suggestion
     * @param named     a {@link Named} config used to name the processor in the topology.
```

Or something similar, it just reads a little bit on the terse side, which I think would confuse people initially. Please also apply this feedback to all the other param docs.